### ISSUE TITLE
Include js file

### ISSUE THREAD

**Issue by aldeed** [meteor-factory/meteor-admin-lte#3]
Is there are reason this doesn't include the adminlte js file? It's needed for some of the plugin features to work, etc.

**Comment by mpowaga** [comment]
Do you mean the [app.js](https://github.com/almasaeed2010/AdminLTE/blob/master/dist/js/app.js) file? We modified it to use blaze events.

**Comment by aldeed** [comment]
Yes. `sidebarSlimScroll` option didn't seem to work until I added that file.
